### PR TITLE
N41: Open Combat

### PR DESCRIPTION
**Pull request by buxx**
Part of https://github.com/rust-gamedev/rust-gamedev.github.io/issues/1233

### CODE REVIEW

File: content/news/041/index.md [hunk 1]
```diff
@@ -69,6 +69,31 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+[Reddit](OpenCombat_reddit))
+by [bux][bux] is a real time tactical game
+which takes place during the 2nd world war.
+
+Since the last news about this game,
+the game engine has been rewritten to permit multiplayer.
+Most of the basic game engine feature have been rewritten with new engine
+(Soldier moves, visibilities, map, etc.).
+Vehicle concept has been introduced to a T-26 tank.
+A basic fight feature like gun fires and shelling have been introduced.
```
Comment (17cupsofcoffee) [inline content/news/041/index.md:90]:
```suggestion
Basic fight features like gunfire and shelling have been introduced.
```

File: content/news/041/index.md [hunk 2]
```diff
@@ -69,6 +69,31 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+[GitHub][OpenCombat_github],
+[Discord][OpenCombat_discord],
+[Reddit](OpenCombat_reddit))
+by [bux][bux] is a real time tactical game
+which takes place during the 2nd world war.
+
+Since the last news about this game,
+the game engine has been rewritten to permit multiplayer.
+Most of the basic game engine feature have been rewritten with new engine
+(Soldier moves, visibilities, map, etc.).
```
Comment (17cupsofcoffee) [inline content/news/041/index.md:88]:
```suggestion
Most of the basic game engine feature have been rewritten with
the new engine (soldier moves, visibilities, map, etc.).
```

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by buxx** [comment]
I resolved the conflict and pushed with your changes @17cupsofcoffee

File: content/news/041/index.md [hunk 3]
```diff
@@ -100,6 +100,31 @@ A more detailed update summary is available [here][de-update-03].
 [de-docs]: https://docs.de-game.org
 [de-update-03]: https://mgn.cz/blog/de03
 
+### [Open Combat]
```
Comment (17cupsofcoffee) [inline content/news/041/index.md:103]:
```suggestion
### [Open Combat][OpenCombat_website]
```

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:fire:

**Comment by ozkriff** [comment]
Thanks!

**Pull request merged**